opentimelineio/adapters/cmx_3600.py

```diff
@@ -104,8 +104,15 @@ def add_clip(self, line, comments, rate=24):
         comment_handler = CommentHandler(comments)
         clip_handler = ClipHandler(line, comment_handler.handled, rate=rate)
         clip = clip_handler.clip
+        reel = clip_handler.reel
+        if reel != 'AX':
+            if not clip.metadata.get('cmx_3600'):
+                clip.metadata.setdefault("cmx_3600", {})
+            clip.metadata['cmx_3600']['reel'] = reel
+
         if comment_handler.unhandled:
-            clip.metadata.setdefault("cmx_3600", {})
+            if clip.metadata.get('cmx_3600') is None:
```

Comment:
Now that I'm looking at this closer, `setdefault` won't kill any initial value.  So I _believe_ that this line and line 109 are unnecessary.

Reply:
We will need to initialize the cmx_3600 dict _somewhere_.
The default in this adapter was to not have that metadata entry unless needed.
I'm adding by default now when we have defined SourceID/Reels, but leaving an exception in to ignore it for AX sources, to preserve some existing assumptions.

I am happy to add it by default for all entries; we will just need to adjust any breaking tests

Reply:
I think what Mike was saying was that `setdefault` is the same as saying: `if clip.metadata.get('cmx_3600'): clip.metadata['cmx_3600'] = {}`

Reply:
so 109/110 can be replaced with the setdefault call, and 114 and 115 can be replaced the same way.  see: https://docs.python.org/2/library/stdtypes.html#dict.setdefault

Reply:
ah, my apologies.  I misread that.
Good catch, I will fix that